ci/build_cuda_parallel_python.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -euo pipefail
+
+# Get the Python version from the command line arguments -py-version=3.10
+py_version=${2#*=}
+echo "Docker socket: " $(ls /var/run/docker.sock)
+
+# given the py_version build the wheel and output the artifact
+# to the artifacts directory
+docker run --rm -i \
+  --workdir /workspace/python/cuda_parallel \
+  --mount type=bind,source=${HOST_WORKSPACE},target=/workspace/ \
+  --env py_version=${py_version} \
+  rapidsai/ci-wheel:cuda12.8.0-rockylinux8-py3.10 \
+  bash -c '\
+    source /workspace/ci/pyenv_helper.sh && \
+    setup_python_env "${py_version}" && \
+    echo "Done setting up python env" && \
+    python -m pip wheel --no-deps /workspace/python/cuda_cccl && \
+    dnf -y install gcc-toolset-13-gcc gcc-toolset-13-gcc-c++ && \
+    echo -e "#!/bin/bash\nsource /opt/rh/gcc-toolset-13/enable" > /etc/profile.d/enable_devtools.sh && \
+    source /etc/profile.d/enable_devtools.sh && \
+    which python && \
+    python -m pip wheel --no-deps . && \
+    python -m pip install patchelf auditwheel && \
```

Comment:
Is `patchelf` needed here explicitly? It is claimed to be a dependency of `auditwheel`.

Reply:
Oh, good  catch - forgot to document this as I was iterating on this PR.

Unfortunately, it looks like `patchelf` is only a test dependency (https://github.com/pypa/auditwheel/blob/c97eb1976730584ceb8d31c6b6ddbeb8278cbf90/pyproject.toml#L44).

And without this line I got some complaints about the system `patchelf` version being too old.

@bdice any chance you have run into that?

Reply:
Not sure. I believe conda-forge has to install its own patchelf.